skbio/alignment/_tabular_msa.py

```diff
@@ -143,6 +144,9 @@ class TabularMSA(MetadataMixin, PositionalMetadataMixin, SkbioObject):
     default_write_format = "fasta"
     __hash__ = None
 
+    read = Read()
```

Comment:
I am thinking if we can further simplify the code by modifying `SkbioObject` such that all classes inheriting from it will have `read` and `write` methods? This may or may not be a good idea though. What do you think?

Reply:
My initial thought is that having the methods explicitly defined in each class is actually a good thing, because it makes it very clear where these methods are coming from. But I will test out putting the read and write descriptors in parent classes and let you know what I find.